src/app_engine/apprtc.py

```diff
@@ -534,8 +534,10 @@ def write_response(self, target_page, params={}):
 
   def get(self):
     """Renders index.html."""
-    if self.request.headers['Host'] == 'apprtc.net':
-      webapp2.redirect('https://www.apprtc.net', permanent=True)
+    REDIRECT_DOMAINS = ['apprtc.appspot.com', 'apprtc.net', 'www.apprtc.net',
+        'apprtc.webrtc.org', 'www.appr.tc']
+    if self.request.headers['Host'] in REDIRECT_DOMAINS:
```

Comment:
do we want to add the path, if one exists? i.e. redirect apprtc.net/r/foo to appr.tc/r/foo